### DISCUSSION TITLE
Wildcard branches for environment variables - "preview/*"

### DISCUSSION THREAD

**Discussion by visormatt** [vercel/community#1236] [Help]
### Summary

Is it currently possible (or in a roadmap) to support using a wildcard branch name like `preview/*` for environment variables. The example / use-case would be having such branches point to a production environment for better load testing. Currently it appears we have to use an `exact match` and existing branch name.

Love the product all around 💯 🤗

**Comment by jadepark-dev** [comment]
I'm also looking for this feature trying to set up CI for branches like `release/{version}`. 
Should I use Vercel API or CLI from CI pipeline to add matches environment variables?

**Reply by thinkjrs** [reply]
I'd also love this feature, specifically for `feature/*` branches (pun intended).

**Comment by SunStupic** [comment]
I want something like this as well. Is it possible to support this?
